tickets/src/components/content/CreateContent.js

```diff
@@ -117,6 +119,18 @@ export class CreateContent extends Component {
     })
   }
 
+  timeChanged(timeDate) {
```

Comment:
Could you add a comment explaining that `TimePicker` handles the internal React plumbing of extracting the data from `target.value`?